lib/msal-common/src/authority/Authority.ts

```diff
@@ -55,6 +58,14 @@ export class Authority {
     protected performanceClient: IPerformanceClient | undefined;
     // Correlation Id
     protected correlationId: string | undefined;
+    // Reserved tenant domain names that will not be replaced with tenant id
+    private static reservedTenantDomains: Set<string> = new Set([
+        Constants.DEFAULT_COMMON_TENANT,
+        "{tenant}",
+        "{tenantid}",
+        "consumers",
```

Comment:
You can use this enum for authority constants FYI: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/eeb136bcb815f32f931fda39d6f9e99d6be29a17/lib/msal-common/src/utils/Constants.ts#LL104C19-L104C19

Reply:
Good point, done.